File: 0645-set-mismatch/0645-set-mismatch.cpp
class Solution {
public:
    vector<int> findErrorNums(vector<int>& nums) {
        int num = -1;
        vector<int> result;
        for(int i = 0; i<nums.size(); i++){
            if(nums[i]==num){
                result.push_back(num);
                result.push_back(num+1);
            }else{
                num = nums[i];
            }
        }
        return result;
    }
};

File: 0645-set-mismatch/0645-set-mismatch.py
class Solution:
    def findErrorNums(self, nums: List[int]) -> List[int]:
        count = Counter(nums)
        ans = [0,0]
        for n in range(1,len(nums)+1):
            if count[n] == 2:
                ans[0] = n
            if count[n] == 0:
                ans[1] = n
        
        return ans